test/functionalTest/cases/1156_qfilters_and_compounds/qfilters_and_compounds_deeper.test

```diff
@@ -653,7 +653,7 @@ Fiware-Correlator: REGEX([0-9a-f\-]{36})
                             "F": false, 
                             "N": null, 
                             "T": true, 
-                            "f": 3.1400000000000001, 
+                            "f": 3.14,
```

Comment:
What about all those 3.14REGEX([01]*) ?
I imagine those can be changed to simply 3.14 now.